lib/imports.js

```diff
@@ -1,13 +1,65 @@
 import {
   createImportDeclaration,
   createNamedImportSpecifier,
-} from './createNode';
+} from "./createNode";
+
+const helpers = {
+  isExclusivelyDefaultImport: (importNode) =>
+    importNode.specifiers.length === 1 &&
+    importNode.specifiers[0].type === "ImportDefaultSpecifier",
+
+  /**
+   * Find an import specifier (import name) matching the moduleName in question.
+   * If a falsy moduleName argument is passed to the parent fn, this test will
+   * be ignored and will return true.
+   * @param   {Node}    importNode
+   * @returns {Boolean}
+   */
+  isMatchingName: (importNode, moduleName) => {
+    if (!moduleName) {
+      return true;
+    }
+
+    return isExclusivelyDefaultImport(importNode)
+      ? importNode.specifiers[0].local.name === moduleName
+      : importNode.specifiers &&
+          importNode.specifiers.find(
+            (specifier) =>
+              specifier.type === "ImportSpecifier" &&
+              specifier.imported &&
+              specifier.imported.name === moduleName
+          );
+  },
+
+  /**
+   * Find an import path matching the importPath in question.
+   * If no importPath argument is passed to the parent fn, this
+   * test will match against `@lendinvest/styleguide`.
+   * If importPath argument is set as `false` to the parent,
+   * this test will be ignored and will always return true.
+   * @TODO remove default to make it platform agnostic
+   * @param   {Node}    importNode
+   * @returns {Boolean}
+   */
+  isMatchingPath: (importNode, importPath) => {
```

Comment:
we need a descriptive name for it, now that it's removed from context. See what other helpers do.
so there's a `isImportMatchedByImportPath` how is it different?